CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # CHANGELOG
 
+### [v2.4 _(Jan 11, 2018)_](https://github.com/omise/omise-magento/releases/tag/v2.5)
+
+### ✨ Highlights
+- Introducing support Installment Payments (#148)
+
+### 🚀 Enhancements
+- Enable Place Order button on Internet Banking checkout option only if bank is selected. (#166)
+- Restrict Tesco Bill Payment to THB transactions only (#165)
+- Tesco Bill Payment added PRINT button (#164)
+- Restrict internet banking payment method to THB orders only(PR #161)
+- Send Tesco barcode to customer's email. (PR #158)
+- Make plugin compatible with PHP 7.2 (PR #168)
+
+### 👾 Bug Fixes
+- Fix for showing Tesco Bill Payment related information on checkout success page when using other payment methods (PR #163).
+- Fix for missing "Terms and Conditions" in Alipay and Tesco payment method. (PR #162)
+- Fixed problem that translation to 'Select a card you want to proceed (PR #160)
+
+### [v2.4 _(Oct 2, 2018)_](https://github.com/omise/omise-magento/releases/tag/2.4)
+
+### ✨ Highlights
+- Save Credit Card (PR #123)
+Now your customer can save credit card details, so they can pay for products more easily the next time the come back to the store.
```

Comment:
@jacstn I think we don't need the description in the CHANGELOG? (Just 1 line of title is enough, or else maybe we can fix the CHANGELOG template a bit if you think we should have PR's description for those highlight tickets)